app/backend/src/couchers/servicers/communities.py

```diff
@@ -203,5 +206,19 @@ def ListEvents(self, request, context):
         return communities_pb2.ListEventsRes()
 
     def ListDiscussions(self, request, context):
-        raise NotImplementedError()
-        return communities_pb2.ListDiscussionsRes()
+        with session_scope() as session:
+            page_size = min(MAX_PAGINATION_LENGTH, request.page_size or MAX_PAGINATION_LENGTH)
+            next_page_id = int(request.page_token) if request.page_token else 0
```

Comment:
Maybe a nicer error message if page_token can't be converted to integer. Can be done later though.

Reply:
I think it's fine like this, it'll be clear from the logs and the exceptions are sanitized anyway at the end. Adding the check everywhere would be annoying and bloat the code, IMO. But we can refactor the page token stuff eventually to have shared helper functions or something, in which case it of course is worth it.

It's (almost certainly) a problem in something we're doing (or someone is poking around on the API) if a non-int ends up here. For the user, it'll just show up as "unknown internal error" or similar.